integrations/pytorch/tutorials/sparse_transfer_learning_tutorial.md

```diff
@@ -0,0 +1,157 @@
+## Downloading and Preparing a Pre-Sparsified Model
+
+First, you need to download the sparsified models from the [SparseZoo](https://sparsezoo.neuralmagic.com/). A few image classification models with [SparseZoo](https://sparsezoo.neuralmagic.com/) stubs:
+
+| Model Name     |      Stub      | Description |
+|----------|-------------|-------------|
+| resnet-pruned-moderate | zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate |This model is a sparse, [ResNet-50](https://arxiv.org/abs/1512.03385) model that achieves 99% of the accuracy of the original baseline model (76.1% top1). Pruned layers achieve 88% sparsity.|
+|resnet-pruned-conservative|zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-conservative|This model is a sparse, [ResNet-50](https://arxiv.org/abs/1512.03385) model that achieves full recovery original baseline model accuracy (76.1% top1). Pruned layers achieve 80% sparsity.|
+|mobilenetv1-pruned-moderate|zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-moderate|This model is a sparse, [MobileNetV1](https://arxiv.org/abs/1704.04861) model that achieves 99% of the accuracy of the original baseline model (70.9% top1). Pruned layers achieve between 70-90% sparsity.|
+|mobilenetv1-pruned-conservative|zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned-conservative|This model is a sparse, [MobileNetV1](https://arxiv.org/abs/1704.04861) model that achieves the same accuracy as the original baseline model. Pruned layers achieve between 60-86% sparsity. This pruned quantized model achieves 70.9% top1 accuracy on the ImageNet dataset.|
+
+Note: The models above were originally trained and sparsified on the [ImageNet](https://image-net.org/) dataset.
+
+- After deciding on which model meets your performance requirements for both speed and accuracy, the following code is used to download the PyTorch checkpoints for the desired model from the SparseZoo:
+```python
+from sparsezoo import Zoo
+    
+PUNED_MODERATE = 'zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate'
+PRUNED_CONSERVATIVE = 'zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-conservative'
+
+stub = PUNED_MODERATE
+model = Zoo.load_model_from_stub(stub)
+downloded_path = model.framework_files[-1].downloaded_path()
+print(f'model with stub {stub} downloaded to {downloded_path}')
+```
+Note the model checkpoint path; this will be used to load the sparsified model.
+
+## Applying a Sparse Transfer Learning Recipe
+
+- Once the model checkpoint has been downloaded, [vision.py](https://github.com/neuralmagic/sparseml/blob/main/integrations/pytorch/vision.py) script can be used to download [Imagenette](https://github.com/fastai/imagenette) and kick-start transfer learning. 
+   The transfer learning process itself is guided using recipes; a minimal example recipe could look like the following:
```

Comment:
I'm not sure if we should include an example recipe in here. If they want to learn more about what the recipes look like, then we should have an external tutorial and the external recipes should contain enough information. This is going to additionally compound once we add in the quantization sparse transfer learning

Reply:
Removed!

Reply:
Okay, removed recipes from tuotrial!